server.js

```diff
@@ -44,6 +52,19 @@ function createApplication(pg_conn_str){
             }
         });
 
+    var sg = require('sendgrid')(process.env.SENDGRID_API_KEY);
+    var request = sg.emptyRequest({
+        method: 'POST',
+        path: '/v3/mail/send',
+        body: mail.toJSON(),
+    });
+
+    sg.API(request, function(error, response) {
```

Comment:
Let's maybe move this out of server.js. How about mailtest.js?

Reply:
yeah, no problem, sorry, I didn't see these messages here until today for some reason